### ISSUE TITLE
Need to improve the accessibility of the table of direct message threads

### ISSUE THREAD

**Issue by fisharebest** [IET-OU/cloudengine#142]
Various issues:
# Possibly as part of fix <<issue 134>> (make whole row of conversation area clickable), each row of the 'table' was split into an individual `<table>` - this messes up what a screen reader speaks.
# Also some missing ALT text.
# Also, missing internationalization.

Obviously a fix for item 1 needs to preserve the functionality and styling of <<issue 134>>!

---
- Bitbucket: https://bitbucket.org/cloudengine/cloudengine/issue/142
- Originally Reported By: [Nick Freear](http://bitbucket.org/nfreear)
- Originally Created At: 2011-02-04T18:28:33.090
